src/Scaler/Services/ScalerMetadata.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Data.Common;
+using Newtonsoft.Json;
+
+namespace Keda.CosmosDb.Scaler
+{
+    [JsonObject(ItemRequired = Required.Always)]
+    internal sealed class ScalerMetadata
+    {
+        private string _metricName;
+
+        public string Connection { get; set; }
+        public string DatabaseId { get; set; }
+        public string ContainerId { get; set; }
+        public string LeaseConnection { get; set; }
+        public string LeaseDatabaseId { get; set; }
+        public string LeaseContainerId { get; set; }
```

Comment:
Wouldn't it be better to introduce a `Lease` object that has these as properties?
```suggestion
        public string LeaseConnection { get; set; }
        public string LeaseDatabaseId { get; set; }
        public string LeaseContainerId { get; set; }
```